provisioner/ansible/test-fixtures/long-debug-message.yml

```diff
@@ -0,0 +1,8 @@
+- name: Stub for Packer testing long Ansible messages
+  hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Very long Ansible output (>65535 chars) (Issue https://github.com/mitchellh/packer/issues/3268)
+      debug:
+        msg: "{{ lipsum(n=300, html=false) }}"
```

Comment:
Ping @cbednarski 

I was a curious about lipsum working in Ansible so I used travis to test multiple versions.

The results are a bit weird. I works ok in Ansible 1.9 and 2.x series but the output message seems to be empty in 1.8 (task does not fail). Do you want me to add a note in the skip test message?